flash/core/model.py

```diff
@@ -143,9 +145,16 @@ def predict(
         """
         running_stage = RunningStage.PREDICTING
         data_pipeline = data_pipeline or self.data_pipeline
+
+        # This check is necessary for mypy compatibility
```

Comment:
does it not work with

`data_pipeline: DataPipeline = data_pipeline or self.data_pipeline`

above?

Reply:
No, because then it complains that it could still be None

Reply:
you can use typing `cast` here